keps/sig-storage/2451-service-account-token-volumes/kep.yaml

```diff
@@ -15,7 +15,7 @@ approvers:
   - "@saad-ali"
 editor: "@zshihang"
 creation-date: 2018-05-15
-last-updated: 2020-03-27
-status: implemented
+last-updated: 2020-04-29
+status: replaced
```

Comment:
probably worth adding a note/link in the README for this as well

Reply:
done